docs/en/reference/how_to_setup_object_storage.md

```diff
@@ -57,6 +57,14 @@ JuiceFS also provides flexible support for this object storage service that dist
 
 Creating a file system using an internal Endpoint ensures better performance and lower latency, and for clients that cannot be accessed through an internal address, you can specify a public Endpoint to mount with the `--bucket` option.
 
+## Storage Class
+
+Object storage usually supports multiple storage classes, such as standard storage, infrequent access storage, and archive storage. When creating an object storage bucket you can choose the appropriate storage class according to your actual needs, or automatically convert the storage class of existing objects through lifecycle management. Regardless of the storage class used, it is supported as the underlying data storage for JuiceFS.
```

Comment:
We can't use the archive (or deep archive) storage class, the objects with that class is not accessible inmediately (requring a restore process).

Reply:
I'll update the document later.

Reply:
Why not do that in this one?

Reply:
Yes, I mean update this PR later.